cpp/basix/e-serendipity.cpp

```diff
@@ -461,12 +416,18 @@ impl::mdarray_t<T, 2> make_serendipity_curl_space_2d(int degree)
             integrand[i] *= pts(i, a);
         }
 
-        wcoeffs(2 * nv + a, psize * d + k)
+        wcoeffs(2 * nv + a, psize * d + nonzero[k])
             = std::reduce(integrand.begin(), integrand.end(), 0.0);
       }
     }
   }
 
+  basix::math::orthogonalise(
+      impl::mdspan_t<T, 2>(
+          wcoeffs.data(),
+          std::array<std::size_t, 2>({wcoeffs.extent(0), wcoeffs.extent(1)})),
```

Comment:
Does `std::array{wcoeffs.extent(0), wcoeffs.extent(1)}` work here?

Reply:
Yes

Reply:
> Yes

Or maybe even `{wcoeffs.extent(0), wcoeffs.extent(1)}` is enough.

Reply:
fails to compile without the std::array

Reply:
I think
```cpp
impl::mdspan_t<T, 2>(wcoeffs.data(), wcoeffs.extent(0), wcoeffs.extent(1));
```
will work.